php/php.dbgp/src/org/netbeans/modules/php/dbgp/ui/DbgpLineBreakpointCustomizerPanel.java

```diff
@@ -184,6 +198,22 @@ public Controller getController() {
         return controller;
     }
 
+    private static int findNumLines(FileObject file) {
+        DataObject dataObject;
+        try {
+            dataObject = DataObject.find (file);
+        } catch (DataObjectNotFoundException ex) {
+            return 0;
+        }
+        EditorCookie ec = (EditorCookie) dataObject.getCookie(EditorCookie.class);
+        if (ec == null) return 0;
+        ec.prepareDocument().waitFinished();
+        Document d = ec.getDocument();
+        if (!(d instanceof StyledDocument)) return 0;
```

Comment:
Please, use better variable names than just `ec` or `d`. Thank you.

Reply:
Done.